conventions/documentation/quick-starts.md

```diff
@@ -0,0 +1,265 @@
+
+### Modals and in-app messaging
+* For confirmation modals included with the instructions, refer to [PatternFly’s modal guidelines](https://www.patternfly.org/v4/components/modal/design-guidelines#get-required-user-input), specifically the “Validate user decisions” section.
+  * **Headline:** A headline is usually phrased as a question. Include important keywords (like “permanent”) in the headline. Avoid extraneous copy, like *Are you sure?*, which distracts users from the main outcome of the action.
+  * **Body text:** Body text gives information about the action’s consequence.
+  * **Buttons:** Buttons allow a user to answer the headline question. These can simply be “yes” or “no,” but adding some context increases clarity—especially when skimming.
+* For error messages, use [PatternFly’s error messaging guidelines](https://www.patternfly.org/v4/ux-writing/error-messages).
+
+### Check your work module
+Once a user completes a step, a Check your work module appears. This module prompts the user to answer a yes/no question about the step results, which gives them the opportunity to review their work. For this module, you’ll only need to write a single yes/no question.
+
+If the user answers Yes, a check mark will appear.
+
+If the user answers No, an error message appears with a link to relevant documentation, if necessary. The user then has the opportunity to go back and try again.
+
+
+## <a name="partiv"></a> Part IV: How can I contribute a quick start?
+
+### Internal contributions
+
+#### Process goals:
+* To provide user-centric, functional quick starts for OpenShift users across a wide range of product areas
+* To create a successful collaboration across teams (UXD, CCS, PM, etc.) for every quick start
+* To define which teams are involved at each step of the quick start effort so that the end result is organized, clean, and in final form for users
+* To respect the bandwidth and input of each team
+
+#### Phase 1: Planning
+1. Determine which product area needs a quick start.
+  * Identify stumbling blocks or complex areas where we can enable user success with as little friction as possible. This information will be gathered from:
+    * UXD user research
+    * Pain points shared from internal feedback sessions (Solution Architects, Technical Marketing, CEE/Customer Support, etc.)
+    * Product Management input
+    * OpenShift or Kubernetes documentation
+    * Features we want to highlight or push more to entice users to try
+    * Areas we want to offer more opinionated views or flows. For example, there are likely cases where there are multiple ways for users to accomplish a task, but we’re recommending a method or suggesting how to get up and running with XYZ.
+  * Note that regardless of where the quick start idea originates, quick starts should always be a part of an epic.
+  * **Team responsible:** Product Management; UXD Design; possibly UXD Research
```

Comment:
These bullets are blending with the rest of the content. Can we add a break before "Team responsible" ?